types/index.d.ts

```diff
@@ -204,8 +228,8 @@ declare namespace RamdaAdjunct {
          * Checks if value is not object-like.
          * A value is object-like if it's not null and has a typeof result of "object".
          */
-        isNotObjLike(val: any): boolean;
-        isNotObjectLike(val: any): boolean; // alias
+        isNotObjLike<T>(val: T): Exclude<T, object>;
+        isNotObjectLike<T>(val: T): Exclude<T, object>; // alias
```

Comment:
You removed explicit information what this symbol is aliasing. Any particular reason for that ?

Reply:
Excuse me, I don't get the question. Could you clarify please?

Reply:
**before:**
```js
// alias of isNotObjLike
```

**now:**
```js
// alias
```

Reply:
No, I didn't change this. [The comment is as is before](https://github.com/char0n/ramda-adjunct/blob/master/types/index.d.ts#L208).

Reply:
No worries, never-mind my comment here. You did not change the comment.